src/commands/statusline.ts

```
@@ -198,6 +230,14 @@ export const statuslineCommand = define({
 		const sessionDisplay = sessionCost != null ? formatCurrency(sessionCost) : 'N/A';
 		const statusLine = `🤖 ${modelName} | 💰 ${sessionDisplay} session / ${formatCurrency(todayCost)} today / ${blockInfo}${burnRateInfo} | 🧠 ${contextInfo ?? 'N/A'}`;
 
+		// Update semaphore before output
+		if (!ctx.values.disableCache) {
```

ESLint

[error] Unexpected any value in conditional. An explicit comparison or type conversion is required.

(ts/strict-boolean-expressions)